src/registry_schemas/schemas/special_resolution.json

```diff
@@ -32,6 +32,17 @@
                 },
                 "signatory": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/person"
+                },
+                "resolutionConfirmed": {
+                    "type": "boolean",
+                    "title": "Confirmation of resolution"
+                },
+                "resolutionText": {
+                    "type": "string",
+                    "maxLength": 1000
+                },
```

Comment:
`resolutionText` already exists as `resolution`.

Reply:
Sure. I'll tell Faiz to change "resolutionText" to "resolution".